### PR TITLE
docs api get detail product

### PR DESCRIPTION
**Pull request by at-trangle**
## Description
docs api get detail product
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: docs/api-references/detail_product.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                },
+                {
+                "id": 31,
+                "product_id": 1,
+                "img_url": "img.jpg",
+                "created_at": "2018-06-08 08:45:50",
+                "updated_at": "2018-06-08 08:45:50",
+                }
+            ],
+            "order_detail": [
```
Comment (at-phuongvu) [inline docs/api-references/detail_product.md]: get `order_detail` làm gì ..

**Comment by at-phuongvu** [comment]
LGTM

File: docs/api-references/detail_product.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+## Product Detail Api
+
+### `GET` Detail Product
+```
+/api/products/{product}
```
Comment (at-quangtran) [inline docs/api-references/detail_product.md]: /api/products/{product} ->/api/products/{id}

File: docs/api-references/detail_product.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Get item product
+#### Request Headers
+| Key | Value | 
+|---|---|
+|Accept|application/json
+
+#### Query Param
+| Key | Value | Description |
+|---|---|---|
+| product | int | Get details of Product (require) |
```
Comment (at-quangtran) [inline docs/api-references/detail_product.md]: | product | -> | id |

File: docs/api-references/detail_product.md (hunk 2 above)
Comment (at-trangle) [reply docs/api-references/detail_product.md]: chỗ này e để theo chuẩn resful à a...theo route:list ạ

**Pull request merged**